frontend/rolecall/src/app/performance/performance-editor.component.ts

```diff
@@ -503,6 +503,10 @@ export class PerformanceEditor implements OnInit, OnDestroy, AfterViewChecked {
     }
     this.intermissions = newInterms;
     if (!this.initCastsLoaded) {
+      for (let s = 0; s < this.state.step_3.segments.length; s++) {
```

Comment:
Let's use `i` instead of `s` for the for-loop index, doesn't look like there's a conflict here and the standard naming is `i`, `j`, and `k` in that order (hopefully no deeper than that).

Reply:
Done.